### PR TITLE
fix: truncate instantiate2 address to have 20 bytes

### PR DESCRIPTION
**Pull request by nseguias**
This PR fixes the length of the address being generated when using instantiate2_address, to have 20 bytes instead of the default 32 bytes.

### CODE REVIEW

File: contracts/injective-auction-pool/src/executions.rs [hunk 1]
```diff
@@ -303,7 +303,6 @@ pub fn settle_auction(
 
     let current_auction_round_response = query_current_auction(deps.as_ref())?;
     let current_auction_round = current_auction_round_response.auction_round;
-    //        .ok_or(ContractError::CurrentAuctionQueryError)?;
```
Comment (gorgos) [inline contracts/injective-auction-pool/src/executions.rs:306]: not sure about this?

File: contracts/injective-auction-pool/src/helpers.rs [hunk 2]
```diff
@@ -120,8 +131,7 @@ pub(crate) fn new_auction_round(
                 );
                 let salt = Binary::from(seed.as_bytes());
 
-                let treasure_chest_address =
-                    Addr::unchecked(instantiate2_address(&checksum, &creator, &salt)?.to_string());
+                let treasure_chest_address = my_address(&checksum, &creator, &salt)?.to_string();
```
Comment (gorgos) [inline contracts/injective-auction-pool/src/helpers.rs:134]: LGTM

**Review by gorgos — APPROVED** [review]

File: contracts/injective-auction-pool/src/executions.rs (hunk 1 above)
Comment (nseguias) [reply contracts/injective-auction-pool/src/executions.rs:306]: this was commented anyway, just removed the comment. Originally I thought this field was optional and built around it

Comment (PFC-developer) [reply contracts/injective-auction-pool/src/executions.rs:306]: this was due to us changing to/from using raw protos to injective-std and back.
there were comments as I hoped we could a upgraded injective-std up quickly and revert the code.

**Pull request merged**